content/english/anti-vegan/no-food-is-fully-vegan.md

```diff
@@ -11,33 +11,103 @@ weight: 5
+> (Argument #1) This argument is an appeal to hypocrisy (*the Tu quoque fallacy*). That I fail to meet my own standards is nothing more than a personal failure and does not invalidate my principles. My failure to pick up a piece of litter that I've dropped does not invalidate my claim that littering is wrong. That I may fall short and in some way be complicit in an animal's death in no way invalidates the moral claim that the exploitation of animals is wrong.
+> 
+> (Argument #2) "It is wrong to inflict unnecessary death on nonhuman animals and it is especially wrong, pro tanto, to do so intentionally. Where one must choose between bringing about death as a side-effect, one should, all else equal, pursue the latter course of action".  
+> - The Intentional Killing of Field Animals and Ethical Veganism *by*  Joe Wills
+> 
+> (Argument #3) "While not logically impossible, it is *almost impossible* to avoid being complicit in suffering with our current food chain. Veganism is a way of living which seeks to exclude, *as far as is possible and practicable*, all forms of exploitation of, and cruelty to, animals [...]. Raising an animal from forcible impregnation to their untimely death merely to satisfy our desire for taste is not comparable to deaths we have little way of avoiding as consumers."  
 > - Definition of veganism verbatim from [The Vegan Society](https://www.vegansociety.com/go-vegan/definition-veganism)
 
 ## Detail - Analytic
 
-### Intention vs Accident
+### Argument #1 *Moral Consistency*
+
+Trying to trip vegans up by exposing real or imagined inconsistencies is common rhetoric. The reality is that vegans (being human) are typically not perfect. Vegans are subject to the constraints of time, money, knowledge, availability of resources, and a million other factors. 
+
+That a vegan may fail in no way invalidates the moral claim that the exploitation of animals is wrong.
+
+> Argument #1 is a fallacy (the appeal to hypocrisy).
+
+### Argument #2 *Intent & Knowledge*
+
+Vegans must eat and, where there are limited options, follow the path of least harm.
+
+
+>A golden rule of moral philosophy is, "all things being equal you ought to choose the action that does the least harm”.
+
+While it is true that some animals do die in the production and harvesting of crops vegans do not ‘accept these deaths’ without question.  Where a crop farmer fails to *exercise due diligence to minimize the harm* and deaths occur, an obligation falls on vegans *as far as is possible and practicable* to seek alternatives (e.g. moving towards organic vegetables, buying from [vegan farms] (https://www.viva.org.uk/what-we-do/vegan-farming), avoiding mono-cultures etc.)
```

Comment:
`[vegan farms] (https://www.viva.org.uk/what-we-do/vegan-farming)` doesn't work as a link, the space needs removing!

Reply:
Weird. Don't have a space (in my local copy) and it works fine (locally).  re-editing it anyway in the hope it fixes....